apollo-router/src/plugins/telemetry/metrics/apollo/list_length_histogram.rs

```diff
@@ -0,0 +1,103 @@
+use std::ops::AddAssign;
+
+use hdrhistogram::Histogram;
+use serde::ser::SerializeSeq;
+use serde::Serialize;
+
+use super::studio::MAX_HISTOGRAM_BUCKETS;
+
+/// A histogram for recording lengths of list fields in GraphQL responses. This implementation is clamped to a maximum
+/// of 386 buckets, a restriction imposed by Studio. The buckets roughly follow order of magnitude of the recorded value,
```

Comment:
I note that duration histogram uses 383 and that was taken from the TS implementation. Which one is correct?

Reply:
I've fixed this in the histogram refactor code.